BranchMetrics/android-branch-deep-linking-attribution
# [Android Automation] Remove dependency on redundant repo

## Reference
SDK-1884 -- [Android Automation] Remove dependency on redundant repo

## Description
Duplicating the qa test app to the SDK repo to reduce an unnecessary cross-repo GHA.
Update the test app to use a local project reference instead of the production Maven to build changes based on most recent commit for PRs.

## Testing Instructions
Clone the repo, see that there is a new gradle config for `Branch-SDK-Automation-TestBed`
Run it and observe it is identical to the original app.

## Risk Assessment [`LOW`]



- [✅ ] I, the PR creator, have tested — integration, unit, or otherwise — this code.

## Reviewer Checklist (To be checked off by the reviewer only)

- Correctness & Style

cc @BranchMetrics/saas-sdk-devs for visibility.
